homeassistant/components/simplisafe/__init__.py

```diff
@@ -296,6 +296,13 @@ def _async_register_base_station(
 ) -> None:
     """Register a new bridge."""
     device_registry = dr.async_get(hass)
+
+    # Check for an old system ID format and remove it:
+    if old_device := device_registry.async_get_device(
+        {(DOMAIN, system.system_id)}  # type: ignore[arg-type]
+    ):
+        device_registry.async_remove_device(old_device.id)
```

Comment:
Should we update the new device with the properties of the old device before removing the old device? Otherwise customizations on the old device are lost.

Reply:
Yes, great call.

Reply:
@MartinHjelmare Is e522cb1 the right way to to this?